Specta/Specta/SPTExcludeGlobalBeforeAfterEach.h

```diff
@@ -3,8 +3,8 @@
  */
 #import <Foundation/Foundation.h>
 
-// This protocol is used for blacklisting classes for global beforeEach and afterEach blocks.
-// If you do not want a class to participate in those just add this protocol to a category and it will be
-// excluded.
+// This protocol was used for blacklisting classes for global beforeEach and afterEach blocks.
+// Now, instead, classes are whitelisted by implementing the SPTIncludeGlobalBeforeAfterEach protocol.
+__deprecated
```

Comment:
Hrm, on a later reflection, I think I'd prefer a single release deprecating it so people will get the message before outright removing. Specta is a fundamental tool for a lot of people, no idea how much this feature is being used.